commit 82b0102650cf85268145a46f0ab488bacf6599a1
Author: Andrew Dunstan <andrew@dunslane.net>
Date:   Wed Jul 3 19:03:31 2013 -0400

    Install all a Makefile's extension controls, not just the first.
    
    Bug introduced by commit 6697aa2bc25c83b88d6165340348a31328c35de6 and
    reported by Robert Haas.

diff --git a/src/makefiles/pgxs.mk b/src/makefiles/pgxs.mk
index f4eaa74..8618aa1 100644
--- a/src/makefiles/pgxs.mk
+++ b/src/makefiles/pgxs.mk
@@ -134,7 +134,7 @@ endif # PROGRAM
 
 installcontrol: $(addsuffix .control, $(EXTENSION))
 ifneq (,$(EXTENSION))
-	$(INSTALL_DATA) $< '$(DESTDIR)$(datadir)/extension/'
+	$(INSTALL_DATA) $^ '$(DESTDIR)$(datadir)/extension/'
 endif
 
 installdata: $(DATA) $(DATA_built)
